.CMake/alg_support.cmake

```diff
@@ -161,39 +161,147 @@ cmake_dependent_option(OQS_ENABLE_SIG_rainbow_Vc_cyclic_compressed "" ON "OQS_EN
 
 option(OQS_ENABLE_SIG_SPHINCS "" ON)
 cmake_dependent_option(OQS_ENABLE_SIG_sphincs_haraka_128f_robust "" ON "OQS_ENABLE_SIG_SPHINCS" OFF)
+if(ARCH STREQUAL "x86_64" AND CMAKE_SYSTEM_NAME MATCHES "" AND OQS_USE_AES_INSTRUCTIONS)
```

Comment:
Could you update the templating code [here](https://github.com/open-quantum-safe/liboqs/blob/master/scripts/copy_from_pqclean/.CMake/alg_support.cmake/add_enable_by_alg.fragment#L7) and [here](https://github.com/open-quantum-safe/liboqs/blob/master/scripts/copy_from_pqclean/.CMake/alg_support.cmake/add_enable_by_alg.fragment#L21) to not check for `CMAKE_SYSTEM_NAME` if `platform['operating_systems']` is not present, and then re-run `copy_from_pqclean.py`?